docs/pages/releases/2021-06-02.mdx

```diff
@@ -175,7 +175,7 @@ postsCount(where: PostWhereInput! = {}): Int
 
 We've updated our Prisma dependency from `2.22.1` to `2.24.0`! Check out the [Prisma release notes](https://github.com/prisma/prisma/releases) for more details.
 
-### Credits <Emoji symbol="Party popper"/>
+### Credits <Emoji symbol="🎉" alt="Party popper"/>
```

Comment:
This one didn't gain attention as it looked almost normal but was missing it's emoji

Reply:
Good catch!